planner_cspace/src/planner_3d.cpp

```diff
@@ -294,8 +295,7 @@ class Planner3dNode
     const auto cb_cost = [this](
         const Astar::Vec& s, const Astar::Vec& e,
         const Astar::Vec& v_start,
-        const Astar::Vec& v_goal) -> float
-    {
+        const Astar::Vec& v_goal) -> float {
```

Comment:
clang-format style looks wrong.

Reply:
I corrected `clang-format` version and it looks fine now.